worlds/noita/Regions.py

```diff
@@ -1,48 +1,51 @@
 # Regions are areas in your game that you travel to.
-from typing import Dict, Set
+from typing import Dict, Set, TYPE_CHECKING
 
-from BaseClasses import Entrance, MultiWorld, Region
+from BaseClasses import Entrance, Region
 from . import Locations
+from .Events import create_all_events
 
+if TYPE_CHECKING:
+    from . import NoitaWorld
+else:
+    NoitaWorld = object
 
-def add_location(player: int, loc_name: str, id: int, region: Region) -> None:
-    location = Locations.NoitaLocation(player, loc_name, id, region)
-    region.locations.append(location)
 
-
-def add_locations(multiworld: MultiWorld, player: int, region: Region) -> None:
+def add_locations(world: NoitaWorld, region: Region) -> None:
     locations = Locations.location_region_mapping.get(region.name, {})
     for location_name, location_data in locations.items():
         location_type = location_data.ltype
         flag = location_data.flag
 
-        opt_orbs = multiworld.orbs_as_checks[player].value
-        opt_bosses = multiworld.bosses_as_checks[player].value
-        opt_paths = multiworld.path_option[player].value
-        opt_num_chests = multiworld.hidden_chests[player].value
-        opt_num_pedestals = multiworld.pedestal_checks[player].value
+        opt_orbs = world.options.orbs_as_checks.value

```

Comment:
i really think this reads better without these extra variable definitions. i also don't see anywhere that the `.value` is actually necessary.